### ISSUE TITLE
levelSetToObj on windows?

### ISSUE THREAD

**Issue by TommyClausner** [TimVanMourik/OpenFmriAnalysis#8]
Hey this issue might be related to #6 - is it in any way possible to use ``levelSetToObj.m`` on Windows? since only precompiled linux and mac versions for the function are available as far as I can see, I was wondering if it would be possible to provide a ``.mexw64`` as well?

Thanks a lot!
Cheers,
Tommy

**Comment by TimVanMourik** [comment]
Yes, that should be possible! As I never work on Windows with MATLAB, I've never gotten round to it. I think it's as simple as running the mex compilation functions from the MATLAB part of: https://github.com/TimVanMourik/LevelSetSource.

If you manage to do that, please send a PR to this repo! :)
